pkgs/development/python-modules/latexify/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, dill
+}:
+
+buildPythonPackage rec {
+  pname = "latexify";
```

Comment:
```suggestion
  pname = "latexify-py";
```

Reply:
Done in bbdd1913fa6474e78da9e0cc2184287c3a8bf0f0